sphinx/tutorial/rose/suites.rst

```diff
@@ -4,8 +4,25 @@
 
 .. _tutorial-rose-suites:
 
-Rose Suite Configurations
-=========================
+Rose Configurations for Workflows
+=================================
+
+.. note::
+
+   The following documentation reflects installing and running a Cylc
+   workflow, and assumes that you have Cylc and the
+   cylc-rose plugin installed.
+
+   To check:
+
+   .. code-block:: bash
+
+      $ cylc version --long
+      8.0.0 (/path/to/install)
+
+      Plugins:
+            cylc-rose       1.0.0   /net/home/h02/tpilling/metomi/cylc-rose
```

Comment:
Should change this to something anonymous.
```suggestion
            cylc-rose       1.0.0   ~/cylc-rose
```

Reply:
oops.
Bit late now, it's in the history.